examples/basics/data_rows.ipynb

```diff
@@ -17,47 +17,100 @@
     "    * Image\n",
     "    * Text\n",
     "    * Video\n",
-    "* A datarow is a member of a dataset \n",
-    "* A datarow cannot exist without belonging to a dataset.\n",
-    "* Datarows are staged to be labeled by attaching the dataset that they are members of to a project\n",
-    "    * See dataset notebook on information about datasets\n",
-    "---\n",
-    "To run this notebook on your own data, set the variables in the next cell"
+    "* A data row is a member of a dataset \n",
```

Comment:
I think you ought to spell `data row` in a uniform way, `data row` or `DataRow`

Reply:
when I say data row I am talking about the concept of a data row in Labelbox. When I say DataRow I am referring to the python object. Does that make sense or should I still just pick one?